crates/egui/src/id.rs

```diff
@@ -45,16 +45,16 @@ impl Id {
 
     /// Generate a new [`Id`] by hashing some source (e.g. a string or integer).
     pub fn new(source: impl std::hash::Hash) -> Id {
-        use std::hash::Hasher;
-        let mut hasher = epaint::ahash::AHasher::default();
+        use std::hash::{BuildHasher, Hasher};
+        let mut hasher = epaint::ahash::RandomState::with_seeds(15, 26, 37, 48).build_hasher();
```

Comment:
Why exactly that seed is used? It is important? It definitely needs a comment. Also it is repeated three times, it is easy to change some of the places accidentally, which can break something, that is non-obvious from the code. I think, a function that will return `RandomState` is needed

Reply:
Let's use a more clear [nothing-up-my-sleeve number](https://en.wikipedia.org/wiki/Nothing-up-my-sleeve_number) like `1, 2, 3, 4`

Reply:
I think, it is also need to mention in a comment, that the selected numbers are just random. Should they need to be the same in all three places? Or it is just a [random values](https://xkcd.com/221/), not connected to each other? What's happened if in the future someone change the one of them to another values?